lib/resty/upstream/healthcheck.lua

```diff
@@ -490,6 +480,28 @@ check = function (premature, ctx)
     end
 end
 
+local function load_upstream_mgr(upstream_mgr)
+    upstream_mgr = upstream_mgr or "ngx.upstreams"
+
+    assert(type(upstream_mgr) == "string" or type(upstream_mgr == "table"),
+           "the upstream_mgr option must be a string (module name) or table (actual module)")
```

Comment:
this line exceed 80 chars, maybe you can use [lua-releng](https://github.com/openresty/nginx-devel-utils/blob/master/lua-releng)